resource-aggregate/service/grpcApi.go

```diff
@@ -201,9 +201,19 @@ func (r RequestHandler) UpdateResource(ctx context.Context, request *commands.Up
 	if err != nil {
 		log.Errorf("cannot publish resource content update events: %v", err)
 	}
+
```

Comment:
Why you don't validate on line 189 if ttl is >= 100ms?

Reply:
Fixed